prime-router/docs/universal-pipeline/route.md

```diff
@@ -1,17 +1,419 @@
+
+*The Translate step will review the Endpoints and remove any observations that are not identified.
+
+
+<table>
+  <tr>
+   <td><strong>Topic</strong>
+   </td>
+   <td>Applies to following topics: <em>full-elr, etor-ti, elr-elims</em>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Operation</strong>
+   </td>
+   <td>Expressions are evaluated with OR operation
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Default</strong>
+   </td>
+   <td><strong>Allow All:</strong> The door is open. No filter is in place
+   </td>
+  </tr>
+</table>
+
+
+
+## Storage
+
+The `Route Function` retrieves messages from the `pdhprodstorageaccount` Azure Storage Account. Within that storage account there is a blob container named `reports` containing folders for use by the Universal Pipeline. The `Convert Function` places all messages into the route folder for retrieval by the `Route Function`. Those messages that match a receiver's filtering will then be placed in the `translate` folder for future retrieval by the `Translate Function`. Messages within the `route` folder are saved to sub-folders equaling the name of the sender.
+
+
+## Filter Reversal
+
+This is a NOT operation on the result of the filters set in the Quality Filter. The primary use is to set the filters for a secondary receiver to ingest all data not accepted by a primary receiver. This may be helpful to keep the qualityFilter setting the same for both the primary and secondary and make it easier to read the configuration.
+
+
+## Logging
+
+
+### Purpose
+
+Filtering logic can be extensive and complex. Recording the outcome of the filters provides internal and external users an important view of events. Logging is particularly important when reports do not pass filtering.
+
+* **Jurisdictional Filter**
+    * results of this filter are <span style="text-decoration:underline;">not</span> logged. Given that most items are only meant for one or maybe a couple jurisdictions out of hundreds, there is little value in logging here.
+* **Other Filters**(Quality, Routing, Processing Mode Code, and Condition)
+    * Following Jurisdictional filtering, all other filter groups use**_ evaluateFilterAndLogResult()_**. Upon failure of a filter in a filter group, the outcome is logged to the ActionLog table
+
+```kotlin
+if (!passes) {
+    val filterToLog = "${
+        if (isDefaultFilter(filterType, filters)) "(default filter) " 
+        else ""
+    }${failingFilterName ?: "unknown"}"
+    logFilterResults(filterToLog, bundle, report, receiver, filterType, focusResource)
+}
+```
+
+* **Code Exceptions**
+    * Results of code exceptions on all filters(including jurisdictional) are logged as warnings in the Action Log table:
```

Comment:
Stay consistent with how we write "Action Log table". Action Log table vs ActionLog table.